### DISCUSSION TITLE
Support for Specialization Trees

### DISCUSSION THREAD

**Discussion by Medriza** [Mezryss/FVTT-Genesys#190] [Ideas]
According to the Road Map, we are still a ways off of the Expanded Players Guide. My assumption is that support for Specialization Trees (a la the FFG SW RPG) will be coming when the EPG support is added. They are obviously added in the unofficial FFGSW Foundry ruleset. I'm not sure how close this project is with those folks, but it works wonderful in their implementation. 

In currently trying to work around setting up some specialization trees, I've found a decent temporary measure using tables with draggable links to the talents. But I ran up against an issue. If I have a Rank 1 Talent on a character, and then drag over another Rank of that talent, it calculates the cost as 10XP, i.e., the cost of a Rank 2 Talent. I've actually created two different copies of the Talent (one with cost of 5XP and another with cost of 15XP respectively), but when I drag the 15XP over, it charges 10XP as it's the 2nd Rank of that Talent. It seems to disregard the level of the talent and its 5XP x level cost.

Since it might be a while before we see Specialization Trees in here, could it be possible to add a flag in the options to do that cost calculation based on what rank it is (how it currently works) or based on the level of the talent being added (how it should work for specialization trees)? This would be a wonderful temporary measure (that hopefully isn't too much trouble) to allow these trees to work in the current implementation.

**Comment by Sergeant-Stutters** [comment]
Did you ever find a decent solution for this? I greatly prefer the Specialization Trees over the basic Talent pyramids.

**Reply by Medriza** [reply]
I'm afraid I did not. I ran a one-shot to test things. But while the basics worked, the system was missing too much to let us do what we needed. Automation like the above was actually getting in the way, so we opted for something else.

**Reply by Sergeant-Stutters** [reply]
> I'm afraid I did not. I ran a one-shot to test things. But while the basics worked, the system was missing too much to let us do what we needed. Automation like the above was actually getting in the way, so we opted for something else.

Yeah, that makes sense. I ended up running into similar frustrations, and right now I'm looking at RPG Sessions as a better alternative. Especially once they add the Data Library feature so that I can create my own custom content to drag and drop on character sheets, that's gonna be my go-to. Thanks for the reply!
